src/Oro/ORM/Query/AST/FunctionFactory.php

```diff
@@ -37,4 +44,19 @@ public static function create($platformName, $functionName, array $parameters)
 
         return new $className($parameters);
     }
+
+    /**
+     * @param string $platformName
+     * @return string
+     */
+    private static function classify($platformName)
```

Comment:
Please, change signature to `classify($word)` as $platformName doesn't represent the parameter purpose